common/cuda_hip/stop/residual_norm_kernels.cpp

```diff
@@ -41,12 +41,12 @@ __global__ __launch_bounds__(default_block_size) void residual_norm_kernel(
     if (tidx < num_cols) {
         if (tau[tidx] <= rel_residual_goal * orig_tau[tidx]) {
             stop_status[tidx].converge(stoppingId, setFinalized);
-            device_storage[1] = true;
+            device_storage[0] = true;
```

Comment:
should we put this into a struct or behind a wrapper to avoid mixing up the two values?